### DISCUSSION TITLE
How to enable zoom with controls based on mouse instead of object?

### DISCUSSION THREAD

**Discussion by jamestouri** [pmndrs/drei#432] [Q&A]
When I try to zoom in and out it zooms to the middle of the camera. Is there away I can get OrbitControls to zoom where the mouse is?

**Comment by MinyoungNa1997** [comment]
I think this is more of a three.js related feature question, maybe ask the question over there?

My guess, according to [Orbitcontrols](https://github.com/mrdoob/three.js/blob/master/examples/jsm/controls/OrbitControls.js) is that the camera move around **target**
```
this.target = new Vector3();
```

Where it's initialized to (0,0,0 ) and updated accordingly
